x-pack/plugins/security_solution/public/common/containers/dashboards/use_security_dashboards_table.tsx

```diff
@@ -19,54 +20,34 @@ export interface DashboardTableItem extends SavedObject<SavedObjectAttributes> {
   description?: string;
 }
 
-const SECURITY_TAG_NAME = 'security' as const;
 const EMPTY_DESCRIPTION = '-' as const;
 
-const getSecurityDashboardItems = async (
-  savedObjectsClient: SavedObjectsClientContract
-): Promise<DashboardTableItem[]> => {
-  if (savedObjectsClient) {
-    const tagResponse = await savedObjectsClient.find<SavedObjectAttributes>({
-      type: 'tag',
-      searchFields: ['name'],
-      search: SECURITY_TAG_NAME,
-    });
-
-    const tagId = tagResponse.savedObjects[0]?.id;
-
-    if (tagId) {
-      const dashboardsResponse = await savedObjectsClient.find<SavedObjectAttributes>({
-        type: 'dashboard',
-        hasReference: { id: tagId, type: 'tag' },
-      });
-
-      return dashboardsResponse.savedObjects.map((item) => ({
-        ...item,
-        title: item.attributes.title?.toString() ?? undefined,
-        description: item.attributes.description?.toString() ?? undefined,
-      }));
-    }
-  }
-  return [];
-};
-
 export const useSecurityDashboardsTableItems = () => {
   const [dashboardItems, setDashboardItems] = useState<DashboardTableItem[]>([]);
-
   const {
     savedObjects: { client: savedObjectsClient },
   } = useKibana().services;
 
   useEffect(() => {
     let ignore = false;
     const fetchDashboards = async () => {
-      const items = await getSecurityDashboardItems(savedObjectsClient);
-      if (!ignore) {
-        setDashboardItems(items);
+      if (savedObjectsClient) {
+        const securityDashboards = await getSecurityDashboards(savedObjectsClient);
```

Comment:
why do the async call if `ignore === true`

Reply:
Well, actually the `ignore` check is meant to prevent the set state if the cleanup function has been executed (if the effect has re-executed for some reason, or if the component has unmounted). I am not sure is worth checking it before the async call, the code is synchronous until that point.
I followed react docs: https://reactjs.org/docs/hooks-faq.html#performance-optimizations (last example)